### ISSUE TITLE
Going back from inside a prefab

### ISSUE THREAD

**Issue by elifer5000** [acoppes/unity-history-window#41]
Thanks for an amazing and useful plugin!

One thing I think would make it even better is being able to go back from a prefab. Say you are looking at the hierarchy and enter inside a prefab. Currently it's not possible to go back (even the history appears greyed out from this point), unless I'm missing something.

**Comment by acoppes** [comment]
Hi, thanks for using it and for the feature request.

Let me check if I understand properly. The case is, you have hierarchy elements enabled, you enter a prefab, select something, then go back to the scene and the prefab elements in the history window can't be selected and in grey color, is that right? and you want to be able to go back to editing the prefab with the inside element selected by clicking in the history window entry?

**Comment by elifer5000** [comment]
Yes, that's correct! It's actually both ways, i.e. being able to go from the scene back to the prefab, or from the prefab back to the scene.
